pkgs/applications/networking/instant-messengers/pidgin/pidgin-plugins/sipe/default.nix

```
@@ -28,6 +28,9 @@ stdenv.mkDerivation rec {
   configureFlags = [ "--without-dbus" ];
   enableParallelBuilding = true;
 
+  # Ignore use of deprecated libxml2 function
+  makeFlags = "CFLAGS=-Wno-error=deprecated-declarations";
```

```suggestion
  env.NIX_CFLAGS_COMPILE = "-Wno-error=deprecated-declarations";
```

---

Checked upstream and there's https://github.com/scibian/pidgin-sipe/blob/1c403e53bdb10c9fe6bc5e466040955f70b154ab/configure.ac#L90

```suggestion
  configureFlags = [
    "--enable-quality-check=no"
   ];
```